packages/pino-log-enricher/tests/versioned/pino.tap.js

```diff
@@ -129,4 +129,30 @@ tap.test('Pino instrumentation', (t) => {
       t.end()
     })
   })
+
+  const configValues = [
+    {
+      name: 'application_logging is not enabled',
+      config: { enabled: false, metrics: { enabled: true } }
+    },
+    {
+      name: 'application_logging.metrics is not enabled',
+      config: { enabled: true, metrics: { enabled: false } }
+    }
+  ]
+  configValues.forEach(({ name, config }) => {
```

Comment:
Hm, Husky didn't catch this? Do we have a problem with the linting config?

Reply:
it's just a warning. husky won't catch warnings as the exit code is not 1.